apisix/plugins/limit-count.lua

```diff
@@ -14,11 +14,12 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 --
-local limit_count = require("apisix.plugins.limit-count.init")
+local fetch_secrets = require("apisix.secret").fetch_secrets
+local limit_count   = require("apisix.plugins.limit-count.init")
 
-local plugin_name = "limit-count"
-local _M = {
-    version = 0.4,
+local plugin_name   = "limit-count"
+local _M            = {
```

Comment:
Why do you add the indent? it's better to keep the original style

Reply:
my bad, seems like my auto-formatter configuration